### PR TITLE
cABJG3Ps finalising landing Page

### PR DESCRIPTION
**Pull request by TuyizeeAnastase**
hey @Ntwali-Josue  can review my landing Page?

### CODE REVIEW

**Comment by ceelogre** [comment]
The branch is called `develop` not `developer`

**Comment by TuyizeeAnastase** [comment]
> The branch is called `develop` not `developer`

Thank you am going to correct it. @ceelogre .

File: contactPage.html [hunk 1]
```diff
@@ -19,8 +19,15 @@ <h1 class="logo">Anastase Tuyizere</h1>
            </ul>
        </nav>
        <div class="contactMe">
-        <h1 class="contact_header">Start Journal with me.</h1>
-       </div>
+        <h1 class="contact_header">Start Journal with me</h1>
```
Comment (Ntwali-Josue) [inline contactPage.html:22]: why are you having two  same h1?

File: index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>MyBrand</title>
+    <title>AnastaseTuyizere</title>
```
Comment (Ntwali-Josue) [inline index.html:6]: Remove one  title here maybe you can remove ` <title>AnastaseTuyizere</title>`!

File: index.html [hunk 3]
```diff
@@ -20,7 +20,7 @@ <h1 class="logo">Anastase Tuyizere</h1>
        </nav>
    </header> 
    <main class="main">
-       <p class="title">Software Engineer and Blogger </p>
+       <p class="title">Software Engineer and Blogger</p>
```
Comment (Ntwali-Josue) [inline index.html:23]: why are you haing 2 same p tags here ?

Comment (TuyizeeAnastase) [reply index.html:23]: thank you get it

File: index.html (hunk 2 above)
Comment (TuyizeeAnastase) [reply index.html:6]: yes, i will remove first title

**Comment by Ntwali-Josue** [comment]
It's develop not developer bro

**Pull request merged**